### PR TITLE
feat:use of credentials,email_verified in userapi

### LINKED ISSUE #89: Make changes in api to allow user with valid email only to login
Api endpoint now can be used by anyone and not only the verified user in random to access and login to the user

### PR DESCRIPTION
**Pull request by amirthapamagar2053**

### CODE REVIEW

File: src/pages/api/user/index.js [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
-    const { email } = req.query;
-    let user = await prisma.user.findUnique({ where: { email: email } });
-    if (!user) {
-      return res.status(204).end();
-    }
-    const userForToken = {
-      email: user.email,
-      id: user.id,
-    };
+    const { token } = req.query;
```
Comment (Serenablack) [inline src/pages/api/user/index.js:8]: is this part of code related to fixing the security flaw during login?

File: src/sections/login/loginPage.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default function LoginPage() {
         phone: "test",
```
Comment (rushilshakya) [inline src/sections/login/loginPage.js]: Please remove "test" phone number

**Review by rushilshakya — CHANGES_REQUESTED** [review]

File: src/sections/login/loginPage.js (hunk 2 above)
Comment (amirthapamagar2053) [reply src/sections/login/loginPage.js]: User model schema has been updated. Phone value is optional

File: src/pages/api/user/index.js (hunk 1 above)
Comment (amirthapamagar2053) [reply src/pages/api/user/index.js:8]: Yes

**Review by rushilshakya — APPROVED** [review]

**Pull request merged**